### PR TITLE
Syntax Highlighting for Visual Studio Code and Atom

### LINKED ISSUE #427: Visual Studio Code syntax highlighting

### PR DESCRIPTION
**Pull request by cyberglot**
Closes #427.

### CODE REVIEW

**Comment by cwgoes** [comment]
What exactly is the procedure to test this - are there instructions somewhere?

File: ide/vsc/syntax-highligthing/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# ju README
```
Comment (adrianbrink) [inline ide/vsc/syntax-highligthing/README.md:1]: How do I install this plugin?

Comment (cyberglot) [reply ide/vsc/syntax-highligthing/README.md:1]: It's still not on the VSC Marketplace, so you cannot install them just yet. I will ping you when I publish it.

**Comment by cyberglot** [comment]
> What exactly is the procedure to test this - are there instructions somewhere?

For VSC, you need to install it, open the folder `juvix/ide/vsc/syntax-highlighting` in VSC and then execute it (second menu item to the left, then click on the green "play" button next to the menu on the left).

For Atom, you need to install it, go to the folder juvix/ide/atom/language-juvix` on your terminal and run `apm link`.

**Comment by cwgoes** [comment]
@thealmarty Have you tested this successfully?

@cyberglot Can we publish the highlighting extension to the store (presumably we can publish updated versions later)?

File: ide/juvix-examples/Id-Strings.ju [hunk 2]
```diff
@@ -0,0 +1,16 @@
+let string = %Michelson.string
```
Comment (thealmarty) [inline ide/juvix-examples/Id-Strings.ju:1]: I think we don't need this file here? There are multiple examples in `test/examples`.

File: ide/vsc/syntax-highligthing/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+This is the README for your extension "ju". After writing up a brief description, we recommend including the following sections.
+
+## Features
+
+Describe specific features of your extension including screenshots of your extension in action. Image paths are relative to this README file.
```
Comment (thealmarty) [inline ide/vsc/syntax-highligthing/README.md:7]: These should be filled out before the release on marketplace I think?

Comment (cyberglot) [reply ide/vsc/syntax-highligthing/README.md:7]: We can update that later too.

**Comment by cyberglot** [comment]
As there's a repo for each of the packages, I'm closing this PR.

**Pull request closed without merging**